docs/admin/provisioning_usb.rst

```diff
@@ -38,5 +39,45 @@ of its airgap:
   again. The formatting process should take only a few seconds.
 - Once formatting is complete, you will need to provide the *Export Device* and
   its decryption password to the SecureDrop Workstation users. Make sure that
-  they store it and its password securely, as it will contain decrypted 
+  they store it and its password securely, as it will contain decrypted
+  submissions.
+
+Creating a VeraCrypt-encrypted drive
+------------------------------------
+
+- If it isn't already done, download and install the `VeraCrypt software <https://www.veracrypt.fr/en/Home.html>`_.
+- Start VeraCrypt from your computer's application or software interface.
+- Click **Create Volume**
+- Select **Encrypt a non-system partition/drive** and click **Next**.
+- Select **Standard VeraCrypt volume** and click **Next**
+- Connect your fresh USB stick and click **Select Device...** to choose your USB.
+
+  - You may see a warning that says "We strongly recommend that inexperienced
+    users create a VeraCrypt file container on the selected device/partition,
+    instead of attempting to encrypt the entire device/partition." Click **Yes**.
+  - Click **Next** to advance.
+
+.. warning:: The formatting operation will wipe any data on an existing partition.
+  Make sure that you select the correct device!
+
+- You will be prompted to set a password. This password
+  should be strong - a 6-word `Diceware <https://en.wikipedia.org/wiki/Diceware>`_
+  passphrase is highly recommended.
+- You will be asked if you need to store large files, select **No** and click **Next**.
+- Select the following options in the Volume Format dialog:
+
+  - Filesystem: FAT
+  - Quick Format: unselected
```

Comment:
Should we enable Quick Format?

Reply:
I haven't tested with it enabled. The only other things I would add here are:
- leave the default PIM settings
- Don't create a hidden VeraCrypt partition

Reply:
No, they should leave it unselected